RNA-seq/02-gastric_cancer_tximport.Rmd

```diff
@@ -32,14 +33,17 @@ library(tximport)
 # directory where the quant files are located, each sample is its own
 # directory
 quant_dir <- file.path("data", "quant", "gastric_cancer")
-# tximport directory
+
+# create a directory to hold the tximport results if it doesn't exist yet
 txi_dir <- file.path("data", "tximport", "gastric_cancer")
```

Comment:
This directory will need to be moved to a non-shared location. `data` is not writeable.

Reply:
To close the loop on this - because we distribute both raw and partially processed data for the bulk RNA-seq module due to time constraints (e.g., "cooking show magic"), we can not make all of `data` not writeable without dramatically changing the module structure. That is out of scope for the virtual pilot. So we will instead symlink individual files and try to find a different solution in the future.